LoRaWAN: Fragmented Data Block Transport Service (#68570)
zephyrproject-rtos/zephyr: samples/subsys/lorawan/fuota/README.rst

.. zephyr:code-sample:: lorawan-fuota
   :name: LoRaWAN FUOTA
   :relevant-api: lorawan_api

>    Demonstration of LoRaWAN firmware-upgrade over the air (FUOTA).

Reviewer: This should be worded as if you were completing the sentence "This code sample shows how to ..."
ex:

```suggestion
   Perform a LoRaWAN firmware-upgrade over the air (FUOTA) operation.
```

Overview
********

An application to demonstrate firmware-upgrade over the air (FUOTA) over LoRaWAN.

The following services specified by the LoRa Alliance are used:

- Application Layer Clock Synchronization (TS003)
- Remote Multicast Setup (TS005)

> - Fragmented Data Block Transport (TS004)

Reviewer: nit: might make sense to link to the specification documents?
